certbot-nginx/certbot_nginx/tests/nginxparser_test.py

```diff
@@ -168,5 +175,58 @@ def test_issue_518(self):
              [['set', '$webp "true"']]]
         ])
 
+class TestUnspacedList(unittest.TestCase):
+    """Test the raw low-level Nginx config parser."""
```

Comment:
Shouldn't this comment say `"""Test the UnspacedList class"""`? That class is only one aspect of the low-level parser and this test doesn't test other parts of the parser.